manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -184,3 +184,14 @@ spec:
       for: 3m
       labels:
         severity: critical
+    - alert: etcdDivergentRevisions
+      annotations:
+        description: etcd is reporting divergent member revisions,
+          with a standard deviation of {{ $value }}.  Members may be
+          stuck or, in extreme circumstances, split brained.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-etcd-operator/FIXME
+        summary: etcd is reporting divergent member revisions.
+      expr: min_over_time(stddev(max by (pod) (etcd_debugging_mvcc_compact_revision{namespace="openshift-etcd"}))[5m:15s]) > 0
```

Comment:
hmm `min_over_time(...) > ...` is probably redundant with `for`.  Maybe this should be:

```yaml
expr: =stddev(max by (pod) (etcd_debugging_mvcc_compact_revision{namespace="openshift-etcd"})) > 0
for: 5m
```

Reply:
I dropped `min_over_time` (and moved to `for: 10m`) with e209966f -> d2ca4071.